pegasus/migrations/83_create_channel_table_metadata.rb

```diff
@@ -0,0 +1,22 @@
+Sequel.migration do
+  up do
+    # TODO - app_id -> channel_id
```

Comment:
Should probably remove this TODO as long as we're okay with doing this in a future PR. Alternatively, I could try to do it now and delay this feature longer (as well as make this PR more complicated). I'm inclined towards just doing this in the future.